perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2328,7 +2328,7 @@ perun_policies:
     include_policies:
       - default_policy
 
-  findCompleteRichMembers_Group_List<String>_List<String>_String_boolean_policy:
+  findCompleteRichMembers_Group_List<String>_List<String>_List<String>_String_boolean_policy:
```

Comment:
Also, please check if any of these policies are used in GUI. We need to make sure that the GUI uses the correct policy names.